### PR TITLE
frontend environment setup

### PR DESCRIPTION
**Pull request by festusebin**

### CODE REVIEW

**Comment by tuzzy08** [comment]
Checks out.

**Review by tuzzy08 — APPROVED** [review]
Good to go from my end.

File: client/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (eagoyi) [inline client/public/index.html:10]: Edit description meta...

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (eagoyi) [inline client/public/index.html:27]: Update Title to Edumentor

**Review by eagoyi — APPROVED** [review]
Kindly effect suggested changes, thanks...

**Pull request closed without merging**